cmd/service/main.go

```diff
@@ -47,6 +48,7 @@ func main() {
 	logger.LogBuildInfo("idmscv-backend")
 	cfg := config.Get()
 	logger.InitLogger(cfg)
+	cfg.Log(*slog.Default())
```

Comment:
Q: About the place to call to this method:

- Do we want to have this information at clowdwatch? or only in the pod? for the second I have one uncertainty: if the pod logs could be recovered if the pod panic during initialization time, before call the config dump.
- Arguments:
  - if the pod logs could be recovered in that case **and** we don't want this in cloudwatch, I would say to move the call just after `cfg := config.Get()`.
  - In other case, the place where it is called I would agree with it.

Reply:
I don't think this is something for blocking the pr and we can think about it later; closing the conversation; if we change our mind we can read the pr and rethink about it.